src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -979,6 +905,72 @@ int ReportCoilSelection::getIndexForOrCreateDataObjFromCoilName(EnergyPlusData &
     return index;
 }
 
+void ReportCoilSelection::associateZoneCoilWithParent(EnergyPlusData &state, std::unique_ptr<CoilSelectionData> &c)
+{
+    c->coilLocation = "Unknown";
+    c->typeHVACname = "Unknown";
+    c->userNameforHVACsystem = "Unknown";
+    // now search equipment
+    auto const &zoneEquipList = state.dataZoneEquip->ZoneEquipList(c->zoneEqNum);
+    bool keepLooking = true;
+    std::string fanType;
+    std::string fanName;
+    for (int equipLoop = 1; equipLoop <= zoneEquipList.NumOfEquipTypes; ++equipLoop) {
+        for (int subEq = 1; subEq <= zoneEquipList.EquipData(equipLoop).NumSubEquip; ++subEq) {
+            if (zoneEquipList.EquipData(equipLoop).SubEquipData(subEq).TypeOf == "FAN:SYSTEMMODEL") {
```

Comment:
I forgot to go back and add in other fan types. VentilatedSlab uses the system fan so I was working out what to do here.

Reply:
@rraustad are you saying you still need to do that in this branch before it marches forward?

Reply:
Yes, and maybe some logic updates when the fan is found.